README.md

```diff
@@ -363,6 +375,12 @@ The following options are supported:
 Dummy network interface, `nm` (NetworkManager) is the only supported `network_provider`
 for this type.
 
+
+
```

Comment:
Suspicious newlines? Can you remove it ?

Reply:
@liangwen12year  the thing is i was to work on issue #238 and issue #292  both are done on README as i try to understand the issues these are documentation issues right? my bad i mad same PR  of them in one branch,i dont know if i miss the real questions of the issues?